lib/l10n/intl_en.arb

```diff
@@ -0,0 +1,63 @@
+{
+  "@@locale": "en",
+  "activityWalking": "Walking",
+  "@activityWalking": {
+    "description": "Walking activity"
+  },
+  "activityRunning": "Running",
+  "@activityRunning": {
+    "description": ""
+  },
```

Comment:
jeżeli description jest puste to usunąłbym, nie jest to wymagane. Chyba, że jest jakiś powód dla którego zdecydowałeś się pisać to wszędzie?

Reply:
Z tego co kojarzę to jest obecnie wymagane. No i zamierzam je uzupełnić.

Reply:
hmm, nie widzę nigdzie info, żeby to było mandatory https://github.com/google/app-resource-bundle/wiki/ApplicationResourceBundleSpecification#predefined-resource-attributes